erts/emulator/beam/bif.tab

```diff
@@ -783,3 +783,11 @@ bif erts_internal:list_to_integer/2
 ubif erlang:min/2
 ubif erlang:max/2
 bif erts_internal:term_to_string/2
+
+#
+# New in 27.
+#
+bif erlang:get_coverage_mode/1
```

Comment:
Adding this to the `erlang` module chafes a bit, it already has a ton of stuff. Have we considered `code:get_coverage(line | function, Module)`?

Reply:
No, I did not consider it, but @michalmuskala did consider moving the functionality to the `code` module in a previous comment.

Reply:
Oops, missed that :sweat_smile: